Marlin/language_it.h

```diff
@@ -174,5 +176,57 @@
 #define MSG_DELTA_CALIBRATE_Y               "Calibra Y"
 #define MSG_DELTA_CALIBRATE_Z               "Calibra Z"
 #define MSG_DELTA_CALIBRATE_CENTER          "Calibra Center"
+#define MSG_INFO_MENU                       "Riguardo stampante"
+#define MSG_INFO_PRINTER_MENU               "Info. stampante"
+#define MSG_INFO_STATS_MENU                 "Statistiche"
+#define MSG_INFO_BOARD_MENU                 "Info. scheda"
+#define MSG_INFO_THERMISTOR_MENU            "Termistori"
+#define MSG_INFO_EXTRUDERS                  "Estrusori"
+#define MSG_INFO_BAUDRATE                   "Baud"
+#define MSG_INFO_PROTOCOL                   "Protocollo"
+#if LCD_WIDTH > 19
+  #define MSG_INFO_PRINT_COUNT              "Contat. stampa"
+  #define MSG_INFO_FINISHED_PRINTS          "Finiti      "
+  #define MSG_INFO_PRINT_TIME               "Tempo totale"
```

Comment:
These 3 strings are meant to align. Please add a trailing space to each of the two the shorter strings.